pmoves/tools/inject_github_pat_from_gh_cli.py

```
@@ -0,0 +1,166 @@
+    if not token:
+        print("ERROR: gh auth token returned empty.", file=sys.stderr)
+        sys.exit(1)
+    return token
+
+
+def get_gh_scopes() -> set[str]:
+    """Return the set of scopes on the current gh CLI token."""
+    try:
+        # `gh auth status` writes scopes to stderr
+        result = subprocess.run(
+            ["gh", "auth", "status"],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)